docs/tutorial/installation.md

```diff
@@ -43,22 +43,75 @@ npm install --platform=win32 electron
 
 If you need to use an HTTP proxy you can [set these environment variables](https://github.com/request/request/tree/f0c4ec061141051988d1216c24936ad2e7d5c45d#controlling-proxy-behaviour-using-environment-variables).
 
+## Custom Mirrors and Caches
+During installation, the `electron` module will call out to [`electron-download`](https://github.com/electron-userland/electron-download) to download prebuilt
+binaries of Electron for your platform. It will do so by contacting GitHub's
+release download page (https://github.com/electron/electron/releases/download/v).
+
+If you are unable to access GitHub or you need to provide a custom build, you
+can do so by either providing mirror or by providing an existing cache directory.
+
+#### Mirror
+You can override the base url, the path at which to look for Electron binaries,
```

Comment:
I think this would be more readable as "You can use environment variables to override the base URL, the path at which to look for Electron binaries, and the binary filename."